README.md

```diff
@@ -96,6 +96,10 @@ This package is currently in the active development.
         # Scopes that you wish to utilize in your application.
         # This should be a simple array of strings.
         scopes:               []
+   
+        # The default role prefix for OAUTH roles that
+        # are generated from the set scopes
+        role_prefix:          'ROLE_OAUTH2_'
```

Comment:
@Allypost @spideyfusion Don't think this was generated with `bin/console config:dump-reference` since the comment is different from the info.

Reply:
@HypeMC Fixed